lib/puppet/parser/functions/consul_sorted_json.rb

```diff
@@ -6,11 +6,13 @@ class << self
 
     def sorted_generate(obj)
       case obj
-        when Fixnum, Float, TrueClass, FalseClass, NilClass
-          return obj.to_json
+        when NilClass
+          return "null"
```

Comment:
Well, but that's not how the current implementation and also how json is specified.

Remember JSON is JavaScript Object Notation and in JavaScript nil is null.

I'm fine with adding the tests that represent/spec the current (pre-this-patch) behavior and hence I will show that the patch does not change it's current behvaior. I will also try to enable travis tests for 1.8.7, so that we could still support this ancient plattform. Yes, I'm very sad to not yet have had the option to move to puppet-server and hence jruby, BUT given that I will need this patch to have the current platform still working properly I thought it makes sense to give such things upstream, as others might run into the same problem on EL6.